### DISCUSSION TITLE
Circumscription of Osmunda

### DISCUSSION THREAD

**Discussion by joelnitta** [pteridogroup/ppg#24] [General]
There were two recent proposals (#9, #10) to recognize nothogenera between segerates of *Osmunda* s.l. The choice to split *Osmunda* s.l. into segregates was made in PPG I (2016), and PPG I did not yet implement nothogenera, so these are logical proposals given PPG I as a starting point. However, the decision of whether to split *Osmunda* s.l. into *Osmunda* s.s., *Plenasium*, and *Claytosmunda* is still worthy of discussion. It was not really discussed in PPG I why that treatment was used. Metzgar et al. 2008 argued against it, whereas a more recent analysis including fossils argued for it (Bomfleur et al. 2017).

Here is a summary from Benjamin Bomfleur received via email about his thinking (posted here with his permission):

> At the very beginning of our project, the Osmunda s.l. classification seemed a perfectly adequate solution to us, too, but following our initial suggestion to separate Osmundastrum (Bomfleur et al., 2015; that was based on the Osmunda pulchella, later Osmundastrum pulchellum), our subsequent comprehensive re-appraisal of the structural and morphological diversity in the group's most informative fossil record, i.e., rhizomes and trunks, really changed my mind. We are confident to have identified the diagnostic significance of the axes' features as good as possible, and based on the character development observable in the many fossils, we trust that the structure of the systematic relationships (arrangement/topology, per se irrespective of rank) especially within Osmundeae (the group comprising all genera still around today) is accurate. Of course, the rank application requires calibration, but the fossil record serves well here, too. Consider how far back in time many of the core lineages especially within Osmundinae can be traced with ease: Claytosmunda shows the full complement of key diagnostic features in both rhizome anatomy and frond morphology already by the Late Triassic (220 Ma!); typical Osmundastrum rhizomes occur in the Triassic as well, and there is good evidence that the extant O cinnamomeum is in existence since the Cretaceous. The unique diagnostic features characteristic of the derived Plenasium show up already in the Cretaceous, and I happen to know of a new, Cretaceous-age O. regalis-like rhizome to be described as the oldest bona fide species record of that genus soon. These particular lineages within modern Osmundaceae are separate for so long; their fossil record testifies to so much greater past diversity (think of the bizarre arborescent Aurealcaulis lineage in Plenasium!) than one would suspect based only on the isolated relictual species... Now, especially in such long-static groups, time passed does not immediately translate into genetic separation, but in this case, I would sure say so. I'll be happy to discuss further!

I just want to point out that the existence of proposals #9 and #10 do not preclude somebody from proposing to use *Osmunda* s.l. If such a proposal were made and passed after the votes on #9 and #10, it would override recognizing the nothogenera. (The current treatment can be changed at any time by passing a proposal, regardless of previous decisions).

Bomfleur B, Grimm GW, Mcloughlin S (2017) The fossil Osmundales ( Royal Ferns )— a phylogenetic network analysis , revised taxonomy , and evolutionary classification of anatomically preserved trunks and rhizomes. https://doi.org/10.7717/peerj.3433

Metzgar JS, Skog J, Zimmer E, Pryer KM (2008) The paraphyly of Osmunda is confirmed by phylogenetic analyses of seven plastid loci. Systematic Botany 33:31–36

**Comment by grimmiges** [comment]
Using the Osmundinae case to implement (in the longer run inevitable) nothogenera makes sense from a principal conceptional point of view, while going back to the _Osmunda_ s.l. concept would sever again our integration of fossil record with molecular-phylogenetic results towards a holistic classification of the group.

I'd like to add that our 2017 adoption of the new generic concept and its integration didn't came out of the blue but was triggered by our re-analyses of Metzgar et al.'s (2008) data set (in Bomfleur et al. 2015; dated in Grimm et al. 2015): the species comprised in the genera as defined in APP I are not only part of the same subtree and unambiguously supported clades, but genetically highly coherent in the studied sequence regions. Unfortunately, Metzgar et al. (like many until the zeroes) only considered the plastid tree topology, the resultant cladogram, but not the substantial molecular evolution and sorting behind it, as e.g. reflected in the according phylogram shown in[ Grimm et al. (2015, fig. 1)](https://academic.oup.com/view-large/figure/115277152/syu108f1.png) They also weren't aware of possibly invevitable and hard-to-test ingroup-outgroup LBA when it comes to define the Osmundaceae root. In all markers _Osmundastrum_ is the most-distinct to all others tips, and naturally attracted by any outgroup tip, all of which are phylogenetically and time-wise very distant from the Osmundaceae. MP cannot escape LBA, and ML only has a 50% chance. I had conversed with Joe Felsenstein on the problem of isolated, genetically diverged ingroup tip(s) and only distant outgroups at hand (we faced a couple of such cases): he said, he wouldn't see any possibility to finally conclude whether such a tip is correctly placed as sister to the rest of the ingroup or because of ingroup-outgroup LBA; we can only try a clock-rooting or filter the gene regions triggering the overly long branches and compare the outcomes. For Metzgar et al.'s data (cf. _Re-visitation of the outgroup-inferred Osmundaceae root_ section in Bomfleur et al. 2015; full data and inferences documentation can be found in our supplements to the 2015 papers), I'm pretty sure their cladogram was biased and our chronogram better reflects the evolutionary history: holophyletic Osmundinae or _Osmunda_ s.l. incl. _Osmundastrum_.

The tree-wise (phylogenetic) and absolute pairwise genetic distance between _Osmundastrum_, _Claytosmunda_ and _Osmunda_ + _Plenasium_ is greater than between the unchallenged sister genera _Todea_ and _Leptopteris_ Even without any fossils and irrespective of where the actual Osmundaceae root is, it makes genetically hence little sense to treat the latter four as members of the same genus, granting the whole Osmundinae the same rank than _Todea_ and _Leptopteris_. The problem with simplistic cladistic classification (based only on outgroup-rooted cladograms) is that clades in rooted trees only give hierarchies of dichotomous splits (assuming a correctly placed ingroup root) but don't quantify the evolution behind them.
PS I would not be surprised if broad-sampling of _Osmundastrum_ and _Osmunda_ s.str. would reveal pseudocryptic or cryptic species.

Our 2015 dating estimates [(Grimm et al. 2015, fig. 2)](https://academic.oup.com/view-large/figure/115277168/syu108f2.png) – using the then novel fossilised birth-death model – are naturally constrained by the well-documented fossil record (based on Bomfleur et al. 2015, the Grimm et al. paper was conceived at least half a year later but published first because we had chosen a more suitable journal when submitting the first draft). The original implementation of the FBD model uses the entirety of the recruited fossil record (backmapped on the chronogram in Grimm et al. 2015, fig. 2) to inform age ranges and model diversification histories for clades; the estimates are either to the point or (somewhat) overestimating in contrast to classic node dating, which per definition only provides minimum divergence estimates and can be very underestimating when relying on an incomplete fossil record (too young node age constraints). But they made at lot of sense on the background of intra-generic genetic coherence – note that the species tips within each genus can be near-identical (highest intrageneric divergence could be found in _Osmunda_ s.str.) – vs. inter-generic divergence observed in Metzgar et al.'s data set, and the spatio-temporal distribution of rhizome phenotypes of each of the modern-day generic lineages and their precursors/extinct ancient siblings (collected in the inevitably paraphyletic fossil genus _Millerocaulis_) as then compiled in Bomfleur et al. (2017).

Thus, the publication of APP I and modification of the generic concept for Osmundaceae came very timely for us, and saved us a lot of taxonomic troubles for the 2017 paper. I would consider the APP I concept with its six modern-day genera a case of consilient or total evidence systematic: a generic concept that considers genetic and morphological divergence patterns, today and their extention into the past, i.e. the evolutionary history of the modern-day genera.

Another purely practical point to consider is that if one goes back to the _Osmunda_ s.l. concept (including, 230+ myrs or not _Osmundastrum_, 150+ myrs), all fossil Osmundinae genera and their species would need to be transferred to _Osmunda_ and all the genera (extant and fossil) would become (again, in case of the modern-day survivors) subgenera. If we extend the genera have to be holophyletic rule into the past, we would also need to include _Todea + Leptopteris_, a monogeneric Osmundaceae (discussed already in [Bomfleur et al. 2015, e.g. fig. 12](https://bmcecolevol.biomedcentral.com/articles/10.1186/s12862-015-0400-7/figures/12), and further in Bomfleur et al. 2017). So, effectively, we would keep the same taxa as one rank below the current concept only to avoid having to implement and recognise a nothogenus. 

I have no access to paywalled literature but assuming there's good genetic evidence for #9 and #10 to erect a nothogenus (ongoing intergeneric hybridisation, see also Wheeler 2014 for extending monophyly to reticulate evolutionary scenarios), I would thus strongly encourage to do so rather than going back to a monogeneric Osmundinae (or Osmundaceae). Note that there was not the slightest sign of intergeneric mixing in Metzgar et al.'s data (per default, I always run single-gene trees in addition to combined tree to fish for obvious ILS or reticulations, the latter requires nuclear data though), which maybe due to uniparental passing of plastomes (I have no idea how chloroplasts get propagated in ferns). More importantly, no evidence incomplete plastid lineage sorting, which is a typical phenomenon associated with ancient or recent hybridisation. So, even if their are inter-generic hybrids, those hybrids seem to have little impact on the species and generic (plastid) gene pools as a whole (Metzgar et al.'s data was a bit disappointing in this aspect, we expected to find more chaos).

Given how ferns propagate and breed, I could imagine that implementing nothogenera will anyhow be inevitable. And the Osmundinae could make a very good case to advocate this need. Given the cytological stability (Bomfleur et al. 2014) and slow evolution in king ferns, occassional inter-generic hybrids are biologically more than thinkable. 

Bomfleur B, McLoughlin S, Vajda V. 2014. Fossilized nuclei and chromosomes reveal 180 million years of genomic stasis in royal ferns. Science 343:1376-1377.

Bomfleur B, Grimm GW, McLoughlin S. 2015. _Osmunda pulchella_ sp. nov. from the Jurassic of Sweden—reconciling molecular and fossil evidence in the phylogeny of modern royal ferns (Osmundaceae). [BMC Evolutionary Biology 15:126.](http://dx.doi.org/10.1186/s12862-015-0400-7)

Grimm GW, Kapli P, Bomfleur B, McLoughlin S, Renner SS. 2015. Using more than the oldest fossils: Dating Osmundaceae with the fossilized birth-death process. [Systematic Biology 64:396–405](https://doi.org/10.1093/sysbio/syu108).

Wheeler WC. 2014. Phyletic groups on networks. Cladistics 40:447–451. Related post by D. Morrison on [_Genealogical World of Phylogenetic Networks_](https://phylonetworks.blogspot.com/2013/11/monophyletic-groups-in-networks.html)
